Srcs/heiftojpeg/heiftojpeg.cpp

```diff
@@ -0,0 +1,226 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <ctype.h>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Magick++.h>
+#include <list>
+#include "hevcimagefilereader.hpp"
+
+using namespace std;
+
+static int VERBOSE = 0;
+static int MAX_SIZE = -1;
+
+static void decodeData(ImageFileReaderInterface::DataVector data, Magick::Image *image)
+{
+    string hevcFileName = "tmp.hevc";
```

Comment:
I'm still concerned that two callers running in the same directory will cause a race condition with static file names. We can work around this using temp directories from Ruby but that seems like it will get forgotten. Maybe the program could use [tmpnam](http://en.cppreference.com/w/cpp/io/c/tmpnam) to work around that.

Reply:
oh right - I lost that suggestion. I will look into this!

Reply:
done with tmpname